node_tree.py

```diff
@@ -457,24 +342,6 @@ def absolute_location(self):
         """ does not return a vactor, it returns a:  tuple(x, y) """
         return recursive_framed_location_finder(self, self.location[:])
 
-
-    def ensure_enums_have_no_space(self, enums=None):
```

Comment:
this exists to support older existiing blend files made before enums where forced to have no spaces. I'm not sure you want to remove this ..